MachineLearning_Engine/__init__.py

```diff
@@ -0,0 +1,21 @@
+#
```

Comment:
Not sure what this file is doing? It's just got the copyright header?

Reply:
Yep, this is the standard way to define a python module - it tells the interpreter where to find functions and classes. Any folder with an `__init__.py` will be scraped.

Reply:
Fair enough :smile: